git-hub

```diff
@@ -580,7 +560,12 @@ def cli():
 @click.option('--label', '-l', default="", help="search PR by label")
 def hub(command, args, user, comment, number, branch, sort, opened, label):
     if command == "pr":
-        pr_num = int(args[0])
+        # if user inputs the pull request number
+        if args[0].isdigit():
+            pr_num = int(args[0])
+        # if user inputs the url of the pull request
+        else:
+            pr_num = int(args[0].split('/')[-1])
```

Comment:
We can be slightly cleverer here, so that you can also paste in URLs like this:

https://github.com/machine-shop/oss_devkit/pull/30/files

We can do that by matching the URL to a regex, something that expects the form: `https://github.com/(org_name)/(repo)/pull/(pr_nr)(/*)`